reinforcement_learning/common/sagemaker_rl/tf_serving_utils.py

```diff
@@ -30,7 +35,7 @@ def export_tf_serving(agent, output_dir):
         input_signature["observations"] = tf.saved_model.utils.build_tensor_info(policy.observations)
 
         output_signature = {}
-        output_signature["actions"] = tf.saved_model.utils.build_tensor_info(policy.sampler)
+        output_signature["actions"] = tf.saved_model.utils.build_tensor_info(policy._sampler)
```

Comment:
Good catch! This seems to depend on the TF version as well. Would it break anything if `policy` is using tf2?

Reply:
So I ran the network_node_compression example after this change (and another, by replacing `logits` with `_action_prob`), and it worked fine.